sdks/python/apache_beam/io/range_trackers.py

```diff
@@ -28,6 +28,25 @@
            'OrderedPositionRangeTracker', 'UnsplittableRangeTracker']
 
 
+class OffsetRange(object):
+
+  def __init__(self, start, stop):
+    self.start = start
+    self.stop = stop
+
+  def split(self, desired_num_offsets_per_split, min_num_offsets_per_split):
+    current_split_start = self.start
+    max_split_size = max(desired_num_offsets_per_split,
+                         min_num_offsets_per_split)
+    while current_split_start < self.stop:
```

Comment:
Wouldn't it be better to divide into equally sized chunks, rather than let the last one be tiny (possibly smaller than min_num_offsets_per_split)?